.gitignore

```diff
@@ -10,3 +10,8 @@
 .overcommit_gems.rb.lock
 *.log
 *sqlite3-journal
+.tool-versions
```

Comment:
I'd prefer not to have these lines in global .gitignore, as they're specific to your development environment. You can put them locally in `.git/info/exclude`.

Reply:
I will remove them. Thanks.